validphys2/src/validphys/scripts/postfit.py

```diff
@@ -140,7 +140,12 @@ def _postfit(results: str, nrep: int, chi2_threshold: float, arclength_threshold
         if not fitdata.check_nnfit_results_path(result_path):
             raise PostfitError('Postfit cannot find a valid results path')
         if not fitdata.check_lhapdf_info(result_path, fitname):
-            raise PostfitError('Postfit cannot find a valid LHAPDF info file')
+            raise PostfitError(
+                'Postfit cannot find a valid LHAPDF info file\n'
+                'Note: Perhaps you forgot to run evolven3fit? Run\n'
+                f'    evolven3fit {result_path.name} {nrep}\n'
```

Comment:
I'm not entirely sure we want to give the user this line because it is not necessarily what they want to run (namely, they might be doing `postfit 100` but they want to do `evolven3fit 150` and reading this msg they might do `evolven3fit 100` then `postfit 100` will fail etc. I think removing this line solves the problem:

1) The people who already know and just forgot will immediately realise and do the correct thing.
2) The people who don't, it is better if they go and look at the docs anyway.
 

```suggestion
```

Reply:
Ok fair enough.